tornettools/parse_onionperf.py

```diff
@@ -159,6 +159,11 @@ def __handle_stream(db, stream, day):
         if goodput is not None:
             db['client_goodput_5MiB'].append(goodput)
 
+        goodput = __goodput_bps(
```

Comment:
FYI I *think* this isn't going to find anything since this code is parsing the public tor data, which doesn't have 10 MB downloads. (It shouldn't hurt anything either, though)

Reply:
Yes did mentioned it above: "Please not that I have added the option to parse the same download size from onionperf. This is not needed at this point since we do not have onionperf clients producing these tests. I have mainly added this for consistency and in case we might consider having a few onionperf clients producing these test in the not so far future."